mswindows/osgeo4w/build_osgeo4w.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/sh
+
+#
+# Usage: build_osgeo4w.sh
+#
+# The following environment variables are supposed to be passed to the build script
+# - SRC: the directory where the grass source code lives
+# - OSGEO4W_ROOT_MSYS: the root directory of OSGeo4W
+# - UNITTEST: If this variable is defined, addition files for unittests are created
+#
+# By default, the script will look for the source code in the current directory
+# and create bin.x86_64-w64-mingw32\grass$ver.bat (run this batch file to start
+# GRASS GIS) and dist.x86_64-w64-mingw32\etc\env.bat.
+#
+# -p	optionally install GRASS GIS to C:\OSGeo4W\opt\grass (run
```

Comment:
`-p` is not moved here. It would still be good to have this option to create an unzippable package for non-admin users.